### ISSUE TITLE
AirDrop to iRead currently works with one book at a time not several books together

### ISSUE THREAD

**Issue by mediter** [zhiyongzou/iRead#11]

**Comment by mediter** [comment]
this is an impressive app already. Apple's Books app is always making arbitrary decision to remove local copies of books that are not read recently even if there is ample storage capacity on the device.

finally, we may have a decent alternative.

**Comment by zhiyongzou** [comment]
That sounds good, but not support now, it will be added to my plan

**Comment by rastaman111** [comment]
yes, an interesting idea by clicking on a chapter in the table of contents, making a move to a boring page
